src/makeCalculator.js

```diff
@@ -1,10 +1,34 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate: (callback, number) => {
+      callback(number);
```

Comment:
The `operate` method currently calls the callback with the `number` argument but does not modify the `result` property of the calculator. Consider updating the `result` based on the callback's operation to make this method functional.